PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--
+     For Work In Progress Pull Requests, please use the Draft PR feature,
+     see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
+
+     For a timely review/response, please avoid force-pushing additional
+     commits if your PR already received reviews or comments.
+
+     Before submitting a Pull Request, please ensure you've done the following:
+     - 👷‍♀️ Create small PRs. In most cases this will be possible.
+     - ✅ Provide tests for your changes.
+     - 📝 Use descriptive commit messages.
+     - 📗 Update any related documentation and include any relevant screenshots.
+-->
```

Comment:
Can we re-think where this goes? I find it a bit cumbersome to move past to get to updating the bit of the template I need to update.

Reply:
I would really like to leave this at the top as it only appears while editing.